common/data_refinery_common/models/experiment.py

```diff
@@ -124,36 +124,46 @@ def to_metadata_dict(self):
         else:
             metadata["source_last_modified"] = ""
 
+        if self.experimentattribute_set.count() > 0:
+            metadata["other_metadata"] = [
+                attribute.to_dict() for attribute in self.experimentattribute_set.all()
+            ]
+
         return metadata
 
     def get_sample_metadata_fields(self):
         """ Get all metadata fields that are non-empty for at least one sample in the experiment.
         See https://github.com/AlexsLemonade/refinebio-frontend/issues/211 for why this is needed.
         """
-        fields = []
-
-        possible_fields = [
-            "sex",
-            "age",
-            "specimen_part",
-            "genotype",
-            "disease",
-            "disease_stage",
-            "cell_line",
-            "treatment",
-            "race",
-            "subject",
-            "compound",
-            "time",
-        ]
+        fields = set()
+
+        possible_fields = set(
```

Comment:
Do we want to keep this this way, or do we want to start using the ontology terms?

Reply:
I think this information is only ever used on the frontend where we want human-readable names.

Reply:
Can we get human readable names from the ontology terms though? I'm just imagining a world where we could list any of the available metadata elements.

Reply:
Yeah, all those associations are stored in the database. Another option would be to return a list/set of `OntologyTerm`s, then let the API render that as the list of dicts `[{'term': <ontology term>, 'name': <human-readable name>}...]` and let the API consumer decide what to do with it.

Reply:
I guess that would break querying though, hmm